setup.py

```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='spotipy',
-    version='2.13.0',
+    version='2.13.1',
```

Comment:
we can keep this as 2.13.0 for now as we will bump it in a specific PR (2.14.0 probably soon)